user-guide/Standard_Apps/PA/Getting_Started_with_PA/Creating_and_Configuring_a_Process_Definition.md

```diff
@@ -0,0 +1,116 @@
+---
+uid: PA_Creating_and_Configuring_a_Process_Definition
+---
+
+# Creating and configuring a process definition
+
```

Comment:
Should users always do all of the procedures listed below, in the order they are listed, for each process definition they add? Could you add a short summary at the top of this page where you clarify this?